src/EditorFeatures/CSharp/InheritanceMargin/CSharpInheritanceMarginService.cs

```diff
@@ -67,6 +68,7 @@ protected override SyntaxToken GetIdentifier(SyntaxNode declarationNode)
                 EventDeclarationSyntax eventDeclarationNode => eventDeclarationNode.Identifier,
                 VariableDeclaratorSyntax variableDeclaratorNode => variableDeclaratorNode.Identifier,
                 TypeDeclarationSyntax baseTypeDeclarationNode => baseTypeDeclarationNode.Identifier,
+                IndexerDeclarationSyntax indexerDeclarationNode => indexerDeclarationNode.ThisKeyword,
```

Comment:
This is not an identifier :)

Reply:
: ) Let me think, ok I would call it GetDeclarationToken()